specification/glossary.md

```diff
@@ -0,0 +1,44 @@
+# Glossary
+
+Below are a list of some OpenTelemetry specific terms that are used across this
+specification.
+
+## (Telemetry) Library
+
+Denotes the actual *OpenTelemetry* library in use.  
```

Comment:
Doesn't have to be OpenTelemetry actually, it could also be a compatible library producing data according to the OTel data format.